mesh/mesh.cpp

```diff
@@ -9577,6 +9577,9 @@ GeometricFactors::GeometricFactors(const Mesh *mesh, const IntegrationRule &ir,
    const int NQ   = ir.GetNPoints();
 
    Vector Enodes(vdim*ND*NE);
+
+   //Support for DG space missing
+
```

Comment:
@v-dobrev , in the case for DG elem_restr is NULL. I was wondering how to handle for that case here?